research-software-directory/RSD-as-a-service
# 130 home page

# Pull request Title

Fixes #130

Changes proposed in this pull request:

*   Create a landing page to replace the current home page

**How to test:**

*   docker-compose down --volume
*   docker-compose build
*   docker-compose up
*   run migrations: cd data-migration && docker-compose up
*   enjoy the new landing page

Pearks:  
it should work now un firefox and without javascript :)

and look something like this:  
![image](https://user-images.githubusercontent.com/4195550/162782802-d92ed983-4988-4d4c-81ab-90c2723bf13e.png)

PR Checklist:
